example/navigators/MainNavigator.tsx

```diff
@@ -21,14 +21,13 @@ export type ExamplesStackNavigatorProps = {
   'Map Types': undefined;
 };
 
-const ExamplesStackNavigator =
-  createStackNavigator<ExamplesStackNavigatorProps>();
+const ExamplesStackNavigator = createStackNavigator<ExamplesStackNavigatorProps>();
 
 export default function MainNavigator() {
   return (
     <ExamplesStackNavigator.Navigator>
       <ExamplesStackNavigator.Screen
-        name="ExamplesListScreen"
+        name='ExamplesListScreen'
```

Comment:
Does prettier config work correctly on your device 🤔? I think we should use `"` instead of `'`

Reply:
I think I might have modify this file before applying commit with prettier settings. Nevertheless thanks for noticing that